services/data-info/src/data_info/services/entry.clj

```diff
@@ -39,11 +38,10 @@
 ;; file specific
 
 (defn- get-file
-  [path]
-  (init/with-jargon (cfg/jargon-cfg) [irods]
-    (if (zero? (item/file-size irods path))
-      ""
-      (ops/input-stream irods path))))
```

Comment:
(this change is mildly unrelated but there wasn't any reason to have the extra with-jargon here so I killed it)